google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -363,6 +367,22 @@ private String getDisplayableVersion() {
   }
 
   private void validateConfiguration() throws ConfigurationException {
+    CloudSdkService sdkService = CloudSdkService.getInstance();
+
+    // Perform cloud sdk validation
+    Path cloudSdkPath = sdkService.getSdkHomePath();
+    Set<CloudSdkValidationResult> results = sdkService.validateCloudSdk(cloudSdkPath);
```

Comment:
On second thought, this might not be worth doing - the java component is only required for AE standard deployments, and up until now, the validations have been treated like more general issues with a cloud sdk installation. 

Right now there isn't a great way for callers to customize which validations they care about. That's something we could consider in the future, but adding that would expand the scope of this PR further than I'd like to.